bin/cqlsh.py

```diff
@@ -603,7 +606,11 @@ def show_version(self):
         # system.Versions['cql'] apparently does not reflect changes with
         # set_cql_version.
         vers['cql'] = self.cql_version
-        print("[cqlsh %(shver)s | Cassandra %(build)s | CQL spec %(cql)s | Native protocol v%(protocol)s]" % vers)
+        if self.scylla_version:
+            vers['build'] = "Scylla {0}".format(self.scylla_version)
```

Comment:
I prefer the .format over this
```
"Scylla %s" % self.scylla_version
```

anyhow the format that was used before here was dict based, and it would be a bit more verbose: 
```
"Scylla %(version)s" % {'version': self.scylla_version}
```

and I would have just use f-strings, but I think I still want to support older versions for now